examples/caching_and_replay/run.py

```diff
@@ -9,12 +11,23 @@
 from cacheablemodel import CacheableSchelling
 
 # As 'replay' is a simulation model parameter in this example, we need to make it available as such
-model_params["replay"] = mesa.visualization.Checkbox("Replay last run?", False)
+model_params["replay"] = mesa.visualization.Checkbox("Replay cached run?", False)
+
+
+def get_cache_file_status(_):
+    """
+    Display an informational text about caching and the status of the cache file (existing versus not existing)
+    """
+    return f"This example writes every simulation run into a cache file. Cached runs can be replayed later. " \
```

Comment:
The description is too long for me, but I think is also fine if you don't want to make it shorter.

Reply:
I made it a little shorter now